docs/config.md

```diff
@@ -252,6 +255,9 @@ The following config options are supported **only by the YAML** config file.
 # YAML config section for each model to be profiled
 profile_models: <comma-delimited-string-list|list|profile_model>
 
+# List of submodels for BLS models
+bls_submodels: <comma-delimited-string-list>
```

Comment:
Submodels is not a terminology that triton uses. Perhaps rename it to `--bls-composing-models`?

Reply:
Thanks for the insight. I've changed to your suggestion.

Reply:
Is submodel a valid term for ensembles? If not, would composing apply there as well?

Reply:
I've changed this already

Reply:
Yes, we use composing for both ensemble and BLS.